libs/sql-schema-describer/src/lib.rs

```diff
@@ -1,5 +1,7 @@
 //! Database description. This crate is used heavily in the introspection and migration engines.
 
+#![deny(rust_2018_idioms, unsafe_code)]
```

Comment:
Why do we have to put the `unsafe_code` warning here?

Reply:
I basically only think of it as a visibility helper: if someone introduces unsafe code, they have to write the unsafe code and remove that lint, so it's going to be more visible. Of course not a big deal, but why not.